### PR TITLE
Add maxFileSize property

### PR DESCRIPTION
**Pull request by Mihaiii**
Based on https://github.com/Starcounter/Images/commit/7424e1bb3f9bbb81d81eb761f2b7b28a2823692e

For https://github.com/Starcounter/HeadsOmni/issues/617 .

### CODE REVIEW

File: starcounter-upload.html [hunk 1]
```diff
@@ -66,6 +66,10 @@
                     type: Number,
                     value: 8192
                 },
+                maxFileSize: {
```
Comment (miyconst) [inline starcounter-upload.html:70]: Please add some comments, describing in which unit the value should be specified, and how to disabled it: use `-1` or `0`.

P.S. I think, the common practice is to use `0` in this case.

File: starcounter-upload.html [hunk 2]
```diff
@@ -344,7 +353,14 @@
                     this.fire("statechange", task);
                     this.processTask(task);
                 }
-            }
+            },
+            checkFile: function (file) {
+                if (file.size > this.maxFileSize && this.maxFileSize > -1) {
+                    alert("File " + file.name + " is too big. Maximum file size is: " + this.maxFileSize + "kb.");
```
Comment (miyconst) [inline starcounter-upload.html]: The [`file.size`](https://developer.mozilla.org/en-US/docs/Web/API/Blob/size) property is specified in bytes, not kilobytes.

Comment (miyconst) [reply starcounter-upload.html]: Using `alert` is evil. Modern browsers usually block alerts, and users will get frustrated without any feedback on why the file is not uploading. I would rather fire the `statechange` event with `fail to upload` state.

**Review by miyconst — CHANGES_REQUESTED** [review]
Please also update readme when adding new properties.

**Comment by Mihaiii** [comment]
@miyconst Addressed, please have another look.
I used `fileUploadError` event because we were already listen for it in the source code of Images app: https://github.com/Starcounter/Images/pull/160/files#diff-6aeb2a46fd0a0e9686c0a490c5b9bdd8R233 .

**Review by miyconst — APPROVED** [review]

**Pull request merged**